pylint/test/functional/arithmetic_operators.py

```diff
@@ -0,0 +1,8 @@
+# pylint: disable=missing-docstring,invalid-name,unused-argument
+def sample_function(*args, **kwargs):
```

Comment:
FWIW pycodestyle (which is what pep8 was renamed to) has E226 disabled by default. I think having it enabled by default in pylint would be a mistake.